flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/ExecutionConfigOptions.java

```diff
@@ -414,6 +415,16 @@
                             "Determines whether CAST will operate following the legacy behaviour "
                                     + "or the new one that introduces various fixes and improvements.");
 
+    @Documentation.TableOption(execMode = Documentation.ExecMode.STREAMING)
+    public static final ConfigOption<Long> TABLE_EXEC_TOPN_CACHE_SIZE =
+            ConfigOptions.key("table.exec.topn.cache-size")
```

Comment:
`topn` is an outdated name. Use `rank` everywhere like `table.exec.rank.cache-size` to be consistent. The old key can be declared as a deprecated key.

Reply:
Thx, please check again!